sanic/mixins/startup.py

```diff
@@ -877,7 +898,8 @@ def serve(
 
             sync_manager.shutdown()
             for sock in socks:
-                sock.shutdown(SHUT_RDWR)
+                if cls.is_socket_connected(sock):
+                    sock.shutdown(SHUT_RDWR)
```

Comment:
Why bother checking and not just wrap this to ignore any exceptions?

Reply:
Yes it make more sense, code updated!